root/static/styles/layout.less

```diff
@@ -604,6 +604,34 @@ span.video {
 /* Don't try to add the video icon on the recording overview tab */
 .tabs .video { display: none; }
 
+span.high-data-quality {
+    font-size: xx-small;
+    padding-right: 5px;
+}
+
+/* unicode large green circle U+1F7E2 */
+span.high-data-quality::before {
+    content: '🟢';
+}
+
+span.low-data-quality {
+    font-size: xx-small;
+    padding-right: 5px;
+}
+
+/* unicode large orange circle U+1F7E0 */
+span.low-data-quality::before {
+    content: '🟠';
+}
+
+/* Don't add the quality icons on the release overview tab */
+.tabs .high-data-quality { display: none; }
+.tabs .low-data-quality { display: none; }
+
+/* Don't add the quality icons on the release page title */
+h1 .high-data-quality { display: none; }
+h1 .low-data-quality { display: none; }
```

Comment:
Would it make sense to have a `showDataQuality` prop on `EntityLink` instead so we can specify that where it should be shown, instead of having a blacklist in the CSS? I guess it depends on whether it's intended to show everywhere other than a few specific cases.

Reply:
Since the first idea of the ticket was "show as much as possible the quality to push people to improve low-quality releases", I think we want to show the icons everywhere except a few places.
I thought about having a new prop on EntityLink but didn't dare since I don't have a good grasp on the react codebase. It would probably be much faster for a core developer to do it than fix my mess :)
Oh and I copied the CSS behavior from the recording 'video' attribute, so the same discussion probably applies to it too.